infrastructure/template.yaml

```
@@ -148,6 +148,21 @@ Globals:
           - SecretArn: !FindInMap [Dynatrace, SecretArn, !Ref Environment]
 
 Resources:
+  LogStreamTrackingTbl:
```

```suggestion
  RedactionLogStreamTrackingTable:
```
More specific name to make it clear it's for redaction records